docs/src/main/sphinx/security/oauth2.rst

```diff
@@ -22,6 +22,10 @@ To enable OAuth 2.0 authentication for Trino, configuration changes are made on
 the Trino coordinator. No changes are required to the worker configuration;
 only the communication from the clients to the coordinator is authenticated.
 
+Set the callback/redirect URL to `https://<trino-coordinator>:<https-port>/oauth2/callback`,
```

Comment:
need to add info about the port not necessarily needed

Reply:
I removed the `<https-port>` part since it's necessary to document it